### PR TITLE
Create wavelength templates for SOAR Goodman 400_SYZY grating

### PR DESCRIPTION
**Pull request by profxj**
As titled.

Handles both the M1 and M2 grating tilts.

### CODE REVIEW

**Comment by profxj** [comment]
Here is the companion PR in the Dev Suite:

https://github.com/pypeit/PypeIt-development-suite/pull/182

**Comment by profxj** [comment]
(base) profx> ./pypeit_test -i soar_goodman_red develop -t 4
Running tests on the following instruments:
    soar_goodman_red

Reducing data from soar_goodman_red for the following setups:
    M1
    M2

Running tests in 4 parallel processes
 0 passed/ 0 failed/ 0 skipped STARTED soar_goodman_red/M2 pypeit (without masters)
 0 passed/ 0 failed/ 0 skipped STARTED soar_goodman_red/M1 pypeit (without masters)
 1 passed/ 0 failed/ 0 skipped PASSED  soar_goodman_red/M2 pypeit (without masters)
 2 passed/ 0 failed/ 0 skipped PASSED  soar_goodman_red/M1 pypeit (without masters)

--- PYPEIT DEVELOPMENT SUITE PASSED 2/2 TESTS (Masters ignored) ---
Testing Started at 2022-01-11T12:34:41.631736
Testing Completed at 2022-01-11T12:42:15.142838
Total Time: 0:07:33.511102

**Review by SunilSimha — CHANGES_REQUESTED** [review]
The code looks alright to me. Here's a reminder to decide whether flexure correction will be incorporated. Also, not sure if you require unit tests but in case you do, here's a reminder for that as well.

**Comment by profxj** [comment]
(base) profx> ./pypeit_test -i soar_goodman_red -t 4 develop
Running tests on the following instruments:
    soar_goodman_red

Reducing data from soar_goodman_red for the following setups:
    M1
    M2

Running tests in 4 parallel processes
 0 passed/ 0 failed/ 0 skipped STARTED soar_goodman_red/M2 pypeit (without masters)
 0 passed/ 0 failed/ 0 skipped STARTED soar_goodman_red/M1 pypeit (without masters)
 1 passed/ 0 failed/ 0 skipped PASSED  soar_goodman_red/M2 pypeit (without masters)
 2 passed/ 0 failed/ 0 skipped PASSED  soar_goodman_red/M1 pypeit (without masters)

--- PYPEIT DEVELOPMENT SUITE PASSED 2/2 TESTS (Masters ignored) ---

**Comment by profxj** [comment]
=============== 318 passed, 4911 warnings in 1644.49s (0:27:24) ================

**Review by SunilSimha — APPROVED** [review]
Looks good to merge.

File: setup.cfg [hunk 1]
```diff
@@ -52,6 +52,8 @@ scripts =
     bin/pypeit_cp_spec1d
 
 [options.extras_require]
+scikit-image =
```
Comment (jhennawi) [inline setup.cfg:55]: Do we still need this? I'm not sure anyone is using it?

**Review by jhennawi — APPROVED** [review]
Please see my comment about the setup.cfg file. After that you can merge.

File: setup.cfg (hunk 1 above)
Comment (profxj) [reply setup.cfg:55]: removing and we'll see if things break

**Pull request merged**